shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/ClusterContextManagerCoordinatorTest.java

```diff
@@ -250,4 +258,63 @@ private OptimizerContext createOptimizerContext() {
+        Collection<ShardingSphereRule> rules = new LinkedList<>();
+        StatusContainedRule mockStatusContainedRule = mock(StatusContainedRule.class);
+        rules.add(mockStatusContainedRule);
+        when(mockShardingSphereRuleMetaData.getRules()).thenReturn(rules);
+        ShardingSphereMetaData mockShardingSphereMetaData = mock(ShardingSphereMetaData.class);
+        when(mockShardingSphereMetaData.getRuleMetaData()).thenReturn(mockShardingSphereRuleMetaData);
+        contextManager.getMetaDataContexts().getMetaDataMap().put("test_schema", mockShardingSphereMetaData);
+        coordinator.renew(mockPrimaryStateChangedEvent);
+        verify(mockStatusContainedRule, times(1)).updateStatus(any());
+    }   
+    
+    @Test
+    public void assertRenewInstanceStatus() {
+        StateEvent mockStateEvent = mock(StateEvent.class);
+        when(mockStateEvent.getInstanceId()).thenReturn(contextManager.getInstanceContext().getInstance().getInstanceDefinition().getInstanceId().getId());
+        Collection<String> testStates = new LinkedList<>();
+        testStates.add(StateType.OK.name());
+        testStates.add(StateType.LOCK.name());
+        when(mockStateEvent.getStatus()).thenReturn(testStates);
+        coordinator.renew(mockStateEvent);
+        assertNotNull(contextManager.getInstanceContext());
+        assertNotNull(contextManager.getInstanceContext().getInstance());
+        assertThat(contextManager.getInstanceContext().getInstance().getStatus(), is(testStates));
+        testStates.add(StateType.CIRCUIT_BREAK.name());
+        coordinator.renew(mockStateEvent);
+        assertThat(contextManager.getInstanceContext().getState().getCurrentState(), is(StateType.CIRCUIT_BREAK));
+    }
+    
+    @Test
+    public void assertRenewWorkerIdChange() {
+        WorkerIdEvent mockWorkerIdEvent = mock(WorkerIdEvent.class);
+        when(mockWorkerIdEvent.getWorkerId()).thenReturn(12223L);
+        when(mockWorkerIdEvent.getInstanceId()).thenReturn(contextManager.getInstanceContext().getInstance().getInstanceDefinition().getInstanceId().getId());
+        coordinator.renew(mockWorkerIdEvent);
+        assertNotNull(contextManager.getInstanceContext());
+        assertNotNull(contextManager.getInstanceContext().getInstance());
+        assertThat(contextManager.getInstanceContext().getWorkerId(), is(12223L));
+    }
+    
+    @Test
+    public void assertRenewInstanceLabels() {
+        LabelsEvent mockLabelsEvent = mock(LabelsEvent.class);
+        Collection<String> labels = new LinkedList<String>();
+        labels.add("test");
+        when(mockLabelsEvent.getLabels()).thenReturn(labels);
+        when(mockLabelsEvent.getInstanceId()).thenReturn(contextManager.getInstanceContext().getInstance().getInstanceDefinition().getInstanceId().getId());
+        coordinator.renew(mockLabelsEvent);
+        assertNotNull(contextManager.getInstanceContext());
+        assertNotNull(contextManager.getInstanceContext().getInstance());
```

Comment:
These 2 assertNotNull maybe useless, the next assertion can include these 2 assertNotNull.

Reply:
+1. I'll update the code